astore/server/auth/factory.go

```diff
@@ -92,9 +94,6 @@ func WithCA(fileContent []byte) Modifier {
 func WithPrincipals(raw string) Modifier {
 	return func(server *Server) error {
 		splitString := strings.Split(raw, ",")
```

Comment:
Use the username as principal? Not in this PR?

Reply:
It's in #61